admin/app/controllers/admin/CommercialController.scala

```diff
@@ -114,21 +114,34 @@ class CommercialController(implicit context: ApplicationContext) extends Control
   def renderInvalidItems() = Action { implicit request =>
     // If the invalid line items are run through the normal extractor, we can see if any of these
     // line items appear to be targeting Frontend.
-    val invalidLineItems = Store.getDfpLineItemsReport().invalidLineItems
+    val invalidLineItems: Seq[GuLineItem] = Store.getDfpLineItemsReport().invalidLineItems
     val invalidItemsExtractor = DfpDataExtractor(invalidLineItems, Nil)
 
+    val advertisers = AdvertiserAgent.get
+    val orders = OrderAgent.get
+    val sonobiAdvertiserId = advertisers.find(_.name.toLowerCase =="sonobi").map(_.id).getOrElse(0L)
```

Comment:
if there wasn't a sonobi advertiser, the sonobi section of the report would display _None_